### PR TITLE
adjust interface to interface to detailed design document

### LINKED ISSUE #1: Wrong interface
Hey there! nice work!
But I think you made a mistake. The interface is wrong.

### PR DESCRIPTION
**Pull request by dneise**
This should fix #1. 

@dianaruthroy in Documentation team, could you please check if we got it right now, before we do the actual implementation?

### CODE REVIEW

**Comment by dianaruthroy** [comment]
Yep looks good!

File: example/idl/example.midl [hunk 1]
```diff
@@ -12,9 +12,9 @@ module example {
 
   interface ExampleComponent : ACS::ACSComponent {
 
-    void set_a_number(in long long a_number);
+    void set(in long long a_number);
 
-    readonly attribute long long a_number;
+    readonly attribute long long value;
```
Comment (dianaruthroy) [inline example/idl/example.midl:17]: I like the way this was clearly phrased. Good job!

**Comment by dneise** [comment]
Why are the tests still passing?
After changing the IDL, the tests should have failed!

**Comment by dianaruthroy** [comment]
Oh .. I just had a look at the tests. 

You made a mistake there ... you never really tested the interface!
I quickly opened another issue (#3) so we do not forget about this.

**Comment by dneise** [comment]
Ah cool .. but I guess we should fix it all together in this PR, no? It does belong together.

**Comment by dianaruthroy** [comment]
Yep .. sounds good to me...

**Comment by dneise** [comment]
Ah perfect .. tests fail as expected .. I am going to finish the implementation now.

**Review by dianaruthroy — APPROVED** [review]

**Pull request merged**